GrowthBook/GrowthBook.cs

```diff
@@ -32,16 +41,45 @@ public GrowthBook(Context context)
             Enabled = context.Enabled;
             Attributes = context.Attributes;
             Url = context.Url;
-            Features = context.Features.ToDictionary(k => k.Key, v => v.Value);
+            Features = context.Features?.ToDictionary(k => k.Key, v => v.Value) ?? new Dictionary<string, Feature>();
             ForcedVariations = context.ForcedVariations;
+
             _qaMode = context.QaMode;
             _trackingCallback = context.TrackingCallback;
             _tracked = new HashSet<string>();
             _assigned = new Dictionary<string, ExperimentAssignment>();
             _subscriptions = new List<Action<Experiment, ExperimentResult>>();
-        }
 
-        // #region Properties
+            var config = new GrowthBookConfigurationOptions
+            {
+                ApiHost = context.ApiHost ?? "https://cdn.growthbook.io",
+                CacheExpirationInSeconds = 60,
+                ClientKey = context.ClientKey,
+                DecryptionKey = context.DecryptionKey,
+                PreferServerSentEvents = true
+            };
+
+            var featureCache = new InMemoryFeatureCache(cacheExpirationInSeconds: 60);
+            var httpClientFactory = new HttpClientFactory(requestTimeoutInSeconds: 60);
+
+            var loggerFactory = context.DefaultLoggerFactory ?? LoggerFactory.Create(builder =>
```

Comment:
Yeah, I think you're right. I'd added in the console/debug logging for testing and provided the DefaultLogLevel property on the Context that could modify this, but looking at it some more I think that's an extraneous property and this can be condensed and not force a particular logging provider on the user. Will fix that. Thanks!